signalfx-transform-helper.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const FIELD_SEPARATOR = '_';
+const MAX_FIELDNAME_LENGTH = 128;
+
+function isPrimitive(x) {
+  return Object(x) !== x;
+}
+
+function sanitize(name) {
+  return name.replace(/[^a-zA-Z0-9\-_]+/gi, '_').substring(0, MAX_FIELDNAME_LENGTH);
```

Comment:
Maybe we should log a warning when sanitized name is longer than the `MAX_FIELDNAME_LENGTH`?

Reply:
done